src/Sylius/Bundle/CoreBundle/Doctrine/ORM/ProductReviewRepository.php

```diff
@@ -26,15 +26,15 @@ class ProductReviewRepository extends EntityRepository implements ProductReviewR
     /**
      * {@inheritdoc}
      */
-    public function findLatestByProductId(string $productId, string $count): array
+    public function findLatestByProductId(int $productId, int $count): array
```

Comment:
I guess product id is mixed

Reply:
Yeah, it is :(

Reply:
Ok, if it's the only comment, please, merge it and I'll fix it in next PR, just don't make it longer 😄

Reply:
All right!

Reply:
OK, unconsciously I've fixed it 🤣  No need to other PR, no chance for free commit 😢